CONTRIBUTING.md

```diff
@@ -365,18 +370,103 @@ Instead, please:
 
 [Back to TOC](#table-of-contents)
 
+### Formatting Documentation
+
+The Kong docs site has a number of features built in to better display content.
```

Comment:
```suggestion
The Kong docs site has a number of features built-in to better display content.
```

https://www.merriam-webster.com/dictionary/built-in

Reply:
It's not a noun or an adjective, but a phrasal verb in this situation. I can rephrase to avoid it, though.